internal/backend/temporalclient/config.go

```diff
@@ -38,6 +38,11 @@ type Config struct {
 }
 
 var (
+	defaultTemporalDevServerConfig = testsuite.DevServerOptions{
+		LogLevel: zapcore.WarnLevel.String(),
+		EnableUI: true,
```

Comment:
will it enable temporal UI in tests as well?
if so, maybe we want to disable UI in tests?

Reply:
I think it should be an **env variable** with default true and set to false in tests mode

Reply:
modes only give you a set of defaults. you can override it using dynamic config. i will disable it for tests though.